.env.example

```
@@ -7,7 +7,8 @@ GOOGLE_API_KEY=              # aistudio.google.com/api-keys
 NVIDIA_API_KEY=              # build.nvidia.com
 
 # Direct providers (optional)
-MINIMAX_API_KEY=              # platform.minimaxi.com (Anthropic-compatible)
+MINIMAX_API_KEY=              # platform.minimaxi.com (China, default) or platform.minimax.io (Global)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)